### DISCUSSION TITLE
Check all folders

### DISCUSSION THREAD

**Discussion by HenkVirtual** [the-djmaze/snappymail#1634] [Ideas]
Maybe I missed something, but our nextcloud snappymail instance does not check folders on schedule base. Therefore we have to refresh the page completely. Is it possible to have a process like we have on the inbox to check all folders too?

**Comment by the-djmaze** [comment]
There is a background process that refresh every 15 minutes.
When it is not working something is hitting the cache and prevents updates.

Please log your network requests in the browser developer tools fto see what is happening.
